src/main/java/emissary/grpc/pool/ConnectionFactory.java

```diff
@@ -85,46 +90,49 @@ protected ConnectionFactory(String host, int port, Configurator configG) {
         this.target = host + ":" + port; // target may be a host or dns service
 
         // How often (in milliseconds) to send pings when the connection is idle
-        this.keepAlive = configG.findLongEntry(GRPC_KEEP_ALIVE_MS, ConnectionDefaults.GRPC_KEEP_ALIVE_MS);
+        this.keepAlive = configG.findLongEntry(GRPC_KEEP_ALIVE_MS, 60000L);
 
         // Time to wait (in milliseconds) for a ping ACK before closing the connection
-        this.keepAliveTimeout = configG.findLongEntry(GRPC_KEEP_ALIVE_TIMEOUT_MS, ConnectionDefaults.GRPC_KEEP_ALIVE_TIMEOUT_MS);
+        this.keepAliveTimeout = configG.findLongEntry(GRPC_KEEP_ALIVE_TIMEOUT_MS, 30000L);
 
         // Whether to send pings when no RPCs are active
         // Note: Seme gRPC services have this set to false and will be noisy if not adjusted
-        this.keepAliveWithoutCalls = configG.findBooleanEntry(GRPC_KEEP_ALIVE_WITHOUT_CALLS, ConnectionDefaults.GRPC_KEEP_ALIVE_WITHOUT_CALLS);
+        this.keepAliveWithoutCalls = configG.findBooleanEntry(GRPC_KEEP_ALIVE_WITHOUT_CALLS, false);
 
         // Max size (in bytes) for incoming messages and message metadata from the server
-        this.maxInboundMessageSize = configG.findIntEntry(GRPC_MAX_INBOUND_MESSAGE_SIZE, ConnectionDefaults.GRPC_MAX_INBOUND_MESSAGE_SIZE);
-        this.maxInboundMetadataSize = configG.findIntEntry(GRPC_MAX_INBOUND_METADATA_SIZE, ConnectionDefaults.GRPC_MAX_INBOUND_METADATA_SIZE);
+        this.maxInboundMessageSize = configG.findIntEntry(GRPC_MAX_INBOUND_MESSAGE_BYTE_SIZE, 1 << 22); // 4 MiB (2^22));
```

Comment:
```suggestion
        this.maxInboundMessageSize = configG.findIntEntry(GRPC_MAX_INBOUND_MESSAGE_BYTE_SIZE, 1 << 22); // 4 MiB (2^22))
```
This trips the "commented out code" lint. Too funny.